sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/TraceHelperTests.cs

```diff
@@ -241,9 +245,11 @@ public void ActivityWithExceptionEventCreatesExceptionTelemetry()
             Assert.Equal(2, telemetryItems.Count());
             Assert.Equal("Exception", telemetryItems[0].Name);
             Assert.Equal("Request", telemetryItems[1].Name);
-            Assert.Equal(exceptionMessage, (telemetryItems[0].Data.BaseData as TelemetryExceptionData).Exceptions.First().Message);
-            Assert.Equal("System.Exception", (telemetryItems[0].Data.BaseData as TelemetryExceptionData).Exceptions.First().TypeName);
-            Assert.Equal("System.Exception: Exception Message", (telemetryItems[0].Data.BaseData as TelemetryExceptionData).Exceptions.First().Stack);
+
+            var telemetryExceptionData = (telemetryItems[0].Data.BaseData as TelemetryExceptionData) ?? throw new Exception("Invalid BaseData");
```

Comment:
is it possible to do Assert.NotNull here?

Reply:
THE LATEST VERSION OF XUNIT HAS THE CORRECT NULLABLE ATTRIBUTES!!! 🎉
I'll update the PR, this is going to fix so many issues!

Reply:
changed to use Assert.NotNull